webapp/graphite/tags/localdatabase.py

```diff
@@ -182,15 +192,20 @@ def list_values(self, tag, valueFilter=None, requestContext=None):
       sql += ' WHERE t.tag=%s'
       params = [tag]
 
-      if valueFilter:
+      if filter:
         # make sure regex is anchored
-        if not valueFilter.startswith('^'):
-          valueFilter = '^' + valueFilter
+        if not filter.startswith('^'):
+          filter = '^' + filter
```

Comment:
shouldn't that filter also be surrounded by `()` to prevent something like `^a|b`

Reply:
yeah fixed that in the rebase